packages/lesswrong/lib/sql/Query.ts

```diff
@@ -19,6 +19,13 @@ class Arg {
       } else {
         this.typehint = "::JSONB[]";
       }
+    } else if (value === null && type instanceof DefaultValueType && type.isNotNull() && type.getDefaultValueString()) {
+      // console.log({type})
```

Comment:
Probably remove this console.log (unless it's still needed for upsert debugging)